src/molecules/Site.h

```diff
@@ -126,7 +127,15 @@ class LJcenter : public Site {
 		Site::readXML(xmlconfig);
 		xmlconfig.getNodeValueReduced("epsilon", _epsilon);
 		xmlconfig.getNodeValueReduced("sigma", _sigma);
-		xmlconfig.getNodeValue("shifted", _shiftRequested);
+		// Read shifted value and check if its boolean (true/false) to avoid legacy-errors
+		std::string strShifted = "false";
+		xmlconfig.getNodeValue("shifted", strShifted);
+		if (strShifted == "true" || strShifted == "false") {
```

Comment:
```suggestion
        std::transform(strShifted.begin(), strShifted.end(), strShifted.begin(), [](auto c) { return std::tolower(c); });
		if (strShifted == "true" || strShifted == "false") {
```
Force the input to lowercase before comparing to avoid annoying input errors.
(Might need an `#include <algorithm>`)